python/instrumentation/openinference-instrumentation-crewai/src/openinference/instrumentation/crewai/_wrappers.py

```diff
@@ -205,15 +212,33 @@ def __call__(
                 ),
             )
             try:
-                response = wrapped(*args, **kwargs)
+                crew_output = wrapped(*args, **kwargs)
+                usage_metrics = crew.usage_metrics
+                if isinstance(usage_metrics, dict):
+                    if "prompt_tokens" in usage_metrics:
+                        span.set_attribute(LLM_TOKEN_COUNT_PROMPT, int(usage_metrics["prompt_tokens"]))
+                    if "completion_tokens" in usage_metrics:
+                        span.set_attribute(LLM_TOKEN_COUNT_COMPLETION, int(usage_metrics["completion_tokens"]))
+                    if "total_tokens" in usage_metrics:
+                        span.set_attribute(LLM_TOKEN_COUNT_TOTAL, int(usage_metrics["total_tokens"]))
+                else:
+                    # version 0.51 and onwards
+                    span.set_attribute(LLM_TOKEN_COUNT_PROMPT, usage_metrics.prompt_tokens)
+                    span.set_attribute(LLM_TOKEN_COUNT_COMPLETION, usage_metrics.completion_tokens)
+                    span.set_attribute(LLM_TOKEN_COUNT_TOTAL, usage_metrics.total_tokens)
```

Comment:
Are we absolutely certain these are set?

Reply:
yea in later versions it's set right before the end of the `kickoff` main function